packages/office-ui-fabric-react/src/components/ContextualMenu/examples/ContextualMenu.Basic.Example.tsx

```diff
@@ -1,79 +1,103 @@
 import * as React from 'react';
-import { ContextualMenuItemType } from 'office-ui-fabric-react/lib/ContextualMenu';
-import { DefaultButton } from 'office-ui-fabric-react/lib/Button';
-import './ContextualMenuExample.scss';
+import { ContextualMenu, ContextualMenuItemType, IContextualMenuItem } from 'office-ui-fabric-react/lib/ContextualMenu';
+
+export interface IContextualMenuBasicExampleState {
+  showContextualMenu?: boolean;
+}
+const menuItems: IContextualMenuItem[] = [
```

Comment:
Nitpicky thing but I've started to put item list definitions like this *after* the main example component, so people viewing the code can more quickly understand the "meat" of how it works. (This is safe from a declaration order perspective as long as you're only using the items within a function/method.)